<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgShadowWidget</class>
 <widget class="QWidget" name="MsgShadowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>634</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1016</width>
    <height>634</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MsgCenterWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QWidget" name="MsgLeftBottomWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>558</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(246, 247, 255);
border-right:1px solid #EBEBEB;
border-bottom-left-radius:8px;
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="labelNoMsg">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>279</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>279</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#labelNoMsg{font: 14px &quot;微软雅黑&quot;;color:#666666;border-bottom-left-radius:0px;border-right:none;};</string>
            </property>
            <property name="text">
             <string>暂无消息记录</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>280</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">QListWidget{border-bottom-left-radius:0px;}
QListWidget::Item{background:#F6F7FF;border-bottom:1px solid #EBEBEB;}
QListWidget::Item:hover{background:#FFFFFF;}
QListWidget::item:selected{background:#FFFFFF;}
QScrollBar::vertical{width:8px;background:rgba(0,0,0,0%);margin:0px,0px,0px,0px;}QScrollBar::handle:vertical{width:8px;background:rgba(0,0,0,25%); border-radius:4px;min-height:20;}QScrollBar::handle:vertical:hover{width:8px;background:rgba(0,0,0,50%); border-radius:4px;min-height:20;}QScrollBar::add-line:vertical{height:9px;width:8px;border-image:url(:/images/a/3.png);subcontrol-position:bottom;}QScrollBar::sub-line:vertical{height:9px;width:8px;border-image:url(:/images/a/1.png);subcontrol-position:top;}QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical{background:rgba(0,0,0,10%);border-radius:4px;};</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="autoScrollMargin">
             <number>16</number>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="MsgLeftTopWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom:1px solid #EBEBEB;
border-right:1px solid #EBEBEB;
background-color: #FAFBFF;
border-top-left-radius:8px;
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>30</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::DefaultContextMenu</enum>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox {
    border: 0px solid gray;
	font: 16px &quot;微软雅黑&quot;;
	color:#333333;
	padding-left:center;
}
QComboBox::drop-down {
    border-left-width: 1px;
	padding-top:5px;
}
QComboBox::down-arrow {
    image: url(:/ZcloudMsgCenter/image/arrow.png);
}

QComboBox QAbstractItemView
{
	border-radius:0px;
	border: 1px solid rgb(161,161,161);
	padding-top:10px;
	padding-bottom:10px;
	outline: 0px;
	
}
QComboBox QAbstractItemView::item {
    height: 30px;
	font: 16px &quot;微软雅黑&quot;;
	color:#333333;
}
 
QListView::item {
    background: white;
	padding-left:center;
}
 
QListView::item:hover {
    background: rgb(245,246,247);
}</string>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>全部消息</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>已读消息</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>未读消息(0)</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="MsgRightTopWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom:1px solid #EBEBEB;
border-top-right-radius:8px;
background-color: rgb(255, 255, 255);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="rightMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QPushButton" name="minPushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>32</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>32</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{border-image: url(:/ZcloudMsgCenter/image/min.png);}
QPushButton:hover,pressed{border-image: url(:/ZcloudMsgCenter/image/min_sel.png);};

</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="maxPushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>32</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>32</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{border-image: url(:/ZcloudMsgCenter/image/max.png);}
QPushButton:hover,pressed{border-image: url(:/ZcloudMsgCenter/image/max_sel.png);};</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="closePushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>32</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>32</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{border-image: url(:/ZcloudMsgCenter/image/close.png);}
QPushButton:hover,pressed{border-image: url(:/ZcloudMsgCenter/image/close_sel.png);};</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="MsgRightBottomWidget" native="true">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-bottom-right-radius:0px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="MsgBtnWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>58</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>58</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-bottom-right-radius:8px;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZcloudMsgCenter.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
